pkg/controller/container-runtime-config/helpers.go

```diff
@@ -440,6 +441,12 @@ func updatePolicyJSON(data []byte, internalBlocked, internalAllowed []string, re
 	if err != nil {
 		return nil, fmt.Errorf("error decoding policy json: %w", err)
 	}
+	scopeRegexp := "^[^/]*(/[^:/]*(/[^:/]*(:[^:/]*)?)?)?$"
+	r, err := regexp.Compile(scopeRegexp)
```

Comment:
Not that this matters but since the regexp is constant in the source I'd personally use `regexp.MustCompile` since it's not reachable as a user/environment error.